hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```diff
@@ -5523,6 +5523,36 @@ objects:
       applyMode: AlwaysApply
       patch: '{"spec":{"channel":"stable-4.9"}}'
       patchType: merge
+- apiVersion: hive.openshift.io/v1
+  kind: SelectorSyncSet
+  metadata:
+    labels:
+      managed.openshift.io/gitHash: ${IMAGE_TAG}
+      managed.openshift.io/gitRepoName: ${REPO_NAME}
+      managed.openshift.io/osd: 'true'
+    name: osd-cluster-acks-4.9
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: 'true'
+      matchExpressions:
+      - key: hive.openshift.io/version-major/minor
```

Comment:
will have to re-run make to fix this

Reply:
There is a github action that would do it but it wont update until @ramonbutter updates the PR :)